playwright.config.ts

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable @typescript-eslint/strict-boolean-expressions */
+
+import { devices } from "@playwright/test";
+import type { PlaywrightTestConfig } from "@playwright/test";
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
```

Comment:
You could probably get rid of this until later (if you end up needing to read from a `.env` file).